File: app/src/main/java/com/savory/login/FacebookButton.java
package com.savory.login;

import android.content.Context;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.util.AttributeSet;
import android.widget.FrameLayout;

import com.savory.R;

public class FacebookButton extends FrameLayout {

    public FacebookButton(@NonNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
        inflate(context, R.layout.button_facebook, this);
    }
}

File: app/src/main/java/com/savory/login/FacebookButton.kt
package com.savory.login

import android.content.Context
import android.content.Intent
import android.util.AttributeSet
import android.view.View
import android.widget.FrameLayout
import com.facebook.FacebookCallback
import com.facebook.login.LoginResult
import com.savory.R

class FacebookButton : FrameLayout {

    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs) {
        View.inflate(context, R.layout.button_facebook, this);
    }
}